Scripts/Items/Functional/SerpentsJawbone.cs

```diff
@@ -0,0 +1,179 @@
+using System;
+using Server.Mobiles;
+using Server.Gumps;
+using System.Collections.Generic;
+using Server.Network;
+using Server.ContextMenus;
+using Server.Multis;
+
+namespace Server.Items
+{
+    public class SerpentsJawbone : Item, ISecurable
+	{
+		public static Dictionary<int, Point3D> Locations { get; set; }
+        private SecureLevel m_SecureLevel;
+     
+		public static void Initialize()
```

Comment:
Can use a static constructor for this, to guarantee initialization before anything in the entire class can be used;

`static SerpentsJawbone( )`